src/loaders/lottie/tvgLottieModel.h

```diff
@@ -1143,6 +1169,7 @@ struct LottieComposition
     Array<LottieFont*> fonts;
     Array<LottieSlot*> slots;
     Array<LottieMarker*> markers;
+    Inlist<LottieSlotData> slotDatas;
```

Comment:
Additional data set looks very inefficient. Couldn't it be associated with `Array<LottieSlot*> slots` ?